.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Add `node_modules`  directory here too as it can be added by accident while setting up linters.